### PR TITLE
Opacity slider issue and increased width of slider

### PR DESCRIPTION
**Pull request by piyushmani**

### CODE REVIEW

File: src/main/webapp/app/entities/flair-bi/modal/modal-tabs/body-properties.component.html [hunk 1]
```diff
@@ -12,9 +12,10 @@
             <label class="control-label flair-model-pad pull-left">Opacity</label>
             <rzslider rz-slider-model="vm.slider.value" rz-slider-options="vm.slider.options"></rzslider>
             <input type="hidden" class="form-control pull-left" ng-model="vm.visual.bodyProperties.opacity">
+
         </div>
-        
-        <div class="row">
+
+        <div class="row"  style="margin-top: 35px;">
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/modal/modal-tabs/body-properties.component.html]: can we move the css to scss file

File: src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js [hunk 2]
```diff
@@ -171,6 +171,9 @@
             vm.propertiesToggled = true;
             vm.sideBarTab = "properties";
             $('#slider').css('display','block'); 
+            $timeout(function() {
+                 $scope.$broadcast('reCalcViewDimensions');
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js]: can we use angular service instead of broadcast

Comment (piyushmani) [reply src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js]: reCalcViewDimensions function is of rz-slider , so we have to broadcast this method when slider become visible.

File: src/main/webapp/app/entities/flair-bi/modal/modal-tabs/body-properties.component.html (hunk 1 above)
Comment (piyushmani) [reply src/main/webapp/app/entities/flair-bi/modal/modal-tabs/body-properties.component.html]: there was no any specific scss file for that , that why i use inline css

File: src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js (hunk 2 above)
Comment (admin-vizcentric) [reply src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js]: @johibkhan2  and @piyushmani  any update on this PR? Can this be merged?

File: src/main/docker/run-all-dependencies.yml [hunk 3]
```diff
@@ -56,7 +56,7 @@ services:
             environment: development
         image: flairbi/flair-registry:latest
         volumes:
-        - path to config folder/flair-github/flair-config:/flair-configuration
```
Comment (johibkhan2) [inline src/main/docker/run-all-dependencies.yml]: please revert this back

File: src/main/docker/run-all-dependencies.yml [hunk 4]
```diff
@@ -56,7 +56,7 @@ services:
...
+        - /home/piyush/code/flair-config:/flair-configuration
```
Comment (johibkhan2) [inline src/main/docker/run-all-dependencies.yml]: this also

**Pull request closed without merging**